client/src/pages/AdminPage/ManageSuggestionsPage/ManageSuggestionsPage.js

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { approveCreateSuggestion, approveUpdateSuggestion, fetchBulkCreateSuggestion, fetchCreateSuggestion,
+    fetchUpdateSuggestion, rejectCreateSuggestion, rejectUpdateSuggestion, rejectBulkCreateSuggestion,
+    approveBulkCreateSuggestion } from '../../../actions/suggestions';
+import Spinner from '../../../components/Spinner/Spinner';
+import ManageSuggestions from '../../../components/AdminPanel/ManageSuggestions/ManageSuggestions';
+import { withRouter } from 'react-router-dom';
+import * as QueryString from 'query-string';
+import { Modal, ProgressBar } from 'react-bootstrap';
+
+class ManageSuggestionsPage extends React.Component {
+
+    static mapStateToProps(state) {
+        return {
+            fetchCreateSuggestion: state.fetchCreateSuggestion,
+            fetchUpdateSuggestion: state.fetchUpdateSuggestion,
+            fetchBulkCreateSuggestion: state.fetchBulkCreateSuggestion,
+            approveCreateSuggestion: state.approveCreateSuggestion,
+            rejectCreateSuggestion: state.rejectCreateSuggestion,
+            approveUpdateSuggestion: state.approveUpdateSuggestion,
+            rejectUpdateSuggestion: state.rejectUpdateSuggestion,
+            approveBulkCreateSuggestion: state.approveBulkCreateSuggestion,
+            rejectBulkCreateSuggestion: state.rejectBulkCreateSuggestion
```

Comment:
This is also not the prettiest, but I'm not really sure how else to handle it.

Reply:
It's fine, any other way of doing it would be unnecessarily complex